src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx

```diff
@@ -27,9 +28,9 @@ function RestartServiceStatusAlert({
     getRestartServiceDetailsQuery,
 }: {
     deployedService: DeployedService;
-    serviceStateRestartQuery: UseMutationResult<string, Error, DeployedService>;
+    serviceStateRestartQuery: UseMutationResult<ServiceOrder, Error, DeployedService>;
```

Comment:
can you please check the type of UseMutationResult.. because DeployedService is not returned by this Mutation request. 
Applies to start, stop, restart status alerts.

Reply:
![image](https://github.com/user-attachments/assets/e39c2755-f30d-4de3-b706-7114ecc447b1)

![image](https://github.com/user-attachments/assets/54e1e4a3-581d-45cf-b768-234a8800a914)

   for UseMutationResult the third params is variables,  if remove, result type will tip error, so it is necessary.